include/wil/result_macros.h

```diff
@@ -664,7 +664,7 @@ WI_ODR_PRAGMA("WIL_FreeMemory", "0")
 
 // Always logs a known failure
 #define LOG_HR(hr)                                              __R_FN(Log_Hr)(__R_INFO(#hr) wil::verify_hresult(hr))
-#define LOG_LAST_ERROR()                                        __R_FN(Log_GetLastError)(__R_INFO_ONLY(nullptr))
+#define LOG_LAST_ERROR()                                        (wil::last_error_context{}, __R_FN(Log_GetLastError)(__R_INFO_ONLY(nullptr)))

```

Comment:
I'm concerned about code bloat. If this is moved into the Log_GetLast* functions the code can be shared.

Reply:
correctness should trump efficiency, so take my comment as a refinement.